utils/build-script-impl

```diff
@@ -2039,6 +2051,15 @@ for host in "${ALL_HOSTS[@]}"; do
                     )
                 fi
 
+                if contains_product "lldb" ; then
```

Comment:
I realise that at the moment that's equivalent (`SKIP_BUILD_LLDB` is inverse of `contains_product "lldb"`). However, what I'm trying to express/check is that lldb is enabled. At the moment there is only one way to disable (`SKIP`), but IMHO it's less fragile to explicitly check if it's enabled (it'll handle another mechanism for disabling lldb, e.g. due to failed dependency).

Also, the return value from `contains_product` was intentional. Returning 0 on success would make the above if statement read:
`if ! contains_product "lldb"`
That looks wrong.

What do you think @rintaro ?